submissions/krylenger/memory-pair-game/style.css

```diff
@@ -0,0 +1,291 @@
+* {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.body {
+    margin: 0px;
+    display: grid;
+    background-color: seashell;
+}
+
+.header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.header__title {
+    background-color: skyblue;
+    text-align: center;
+    width: 100%;
+    height: 30px;
+    padding: 10px 0px;
+    font-size: 30px;
+    font-weight: 500;
+    padding-left: 150px;
+}
+
+.header__title--subtitle {
+    background-color: yellow;
+    font-size: 27px;
+}
+
+.main {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    margin-top: 30px;
+}
+
+.main__grid-container {
+    display: grid;
+    grid-template: repeat(4, 150px) / repeat(5, 150px);
+    grid-gap: 10px;
+    padding: 20px 40px;
+}
+
+.card {
+    perspective: 1000px;
+    font-size: 20px;
+}
+
+.card--0 {
+    grid-column: 1;
+    grid-row: 4;
+}
+
+.card--1 {
+    grid-column: 2;
+    grid-row: 3;
+}
+
+.card--2 {
+    grid-column: 2;
+    grid-row: 2;
+}
+
+.card--3 {
+    grid-column: 2;
+    grid-row: 1;
+}
+
+.card--4 {
+    grid-column: 3;
+    grid-row: 1;
+}
+
+.card--5 {
+    grid-column: 4;
+    grid-row: 1;
+}
+
+.card--6 {
+    grid-column: 5;
+    grid-row: 1;
+}
+
+.card--7 {
+    grid-column: 5;
+    grid-row: 2;
+}
+
+.card--8 {
+    grid-column: 5;
+    grid-row: 3;
+}
+
+.card--9 {
+    grid-column: 4;
+    grid-row: 4;
+}
+
+.card--background-green {
+    background-color: palegreen;
+}
+
+.card--orel-img {
+    background-image: url('images/orel.png');
+    background-size: cover;
```

Comment:
You added `background-size: cover;` to all cards. 
You can also set this style for some common element and set the `background-image` for the cards wit h unique classes

Reply:
✅ Done